templates/podcasts.html

```diff
@@ -2,14 +2,19 @@
 {% block content %}
 {% set matrix_live = load_data(path="partials/podcasts/matrix_live.md") %}
 {% set otwsu = load_data(path="partials/podcasts/otwsu.md") %}
-<div class="content">
+<div>
     <header>
     </header>
-    <div class="podcasts-grid">
-        <div>
+    <div class="podcasts-content">
+        <div class="podcasts-column">
             {{ matrix_live | markdown | safe }}
+            <div class="badges">
+                <a href="https://podcasts.apple.com/gb/podcast/matrix-live/id1498631706"><img src="/assets/apple_podcasts_badge.svg" alt="Listen on Apple Podcasts" /></a>
+                <a href="https://open.spotify.com/show/1SFrZPISzYs0elxlZUAvZ5?si=Xyns2Pl0RBuGBjuzH4yxsQ"><img src="/assets/spotify_podcasts_badge.svg" alt="Listen on Spotify" /></a>
+                <a href="https://pca.st/uv1qytez"><img src="/assets/pocketcasts_badge.svg" alt="Listen on Pocket Casts" /></a>
+            </div>
```

Comment:
Why is this part of the template rather than part of the content?

Reply:
Hm, do you suggest to add markdown to the matrix_live partial or to put the podcasting platform links into a toml file (with path to image, link, and alt text), or to create an additional partial for badges?

Reply:
I was suggesting to add markdown to the partial.

Reply:
I'm not sure how we could put the platform links in a div, if it was part of the partial content? There's a bit of CSS for this div in particular, to make spacing and wrapping look correct

Reply:
You can write HTML in markdown. You *might* have to put in extra newlines for the markdown img / link syntax to work, as in

```md
<div class="badges">

[![Listen on Apple Podcasts](/assets/apple_podcasts_badge.svg)](https://podcasts.apple.com/gb/podcast/matrix-live/id1498631706)
<!-- other two links ->

</div>
```

or define a shortcode like

```md
{% div(class="badges") %}
[![Listen on Apple Podcasts](/assets/apple_podcasts_badge.svg)](https://podcasts.apple.com/gb/podcast/matrix-live/id1498631706)
<!-- other two links ->
{% end %}

Reply:
I had to use shortcodes to generate the `<a><img /></a>`, zola would choke on image in link markdown